src/components/FilterTop.js

```diff
@@ -84,14 +87,23 @@ function Filter(props) {
   const [tag, setTag] = useState(null);
   const [tagSearchString, setTagSearchString] = useState('');
   const [organizationId, setOrganizationId] = useState(ALL_ORGANIZATIONS);
+  const [userHasOrg, setUserHasOrg] = useState(false);
+  const [orgList, setOrgList] = useState(
+    props.organizationState.organizationList || [],
+  );
 
   useEffect(() => {
     props.tagsDispatch.getTags(tagSearchString);
   }, [tagSearchString, props.tagsDispatch]);
 
   useEffect(() => {
-    props.organizationDispatch.loadOrganizations();
-  }, [props.organizationDispatch]);
+    log.debug('filter top checks user org id & loads orgs');
+    const hasOrg = getOrganization();
+    setUserHasOrg(hasOrg ? true : false); // check if it's an org account or admin
+    if (!hasOrg) {
+      setOrgList(props.organizationState.organizationList); // only load if it's not an org account
```

Comment:
We're still erroneously loading the full org list into the client, just not into the `FilterTop` component.

Reply:
Working to resolve on the server...

Reply:
I left this state setup because I believe it may be helpful when we want to include/exclude sub organizations in future.